src/commands/showFile.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { extname } from "path";
+import { TextDocument, window, workspace } from "vscode";
+import { getFile, IFileResult } from "vscode-azureappservice";
+import { FileEditor } from "../explorer/editors/FileEditor";
+import { FileTreeItem } from "../explorer/FileTreeItem";
+
+export async function showFile(node: FileTreeItem, fileEditor: FileEditor): Promise<void> {
+    // tslint:disable-next-line:strict-boolean-expressions
+    if (!node.lastClick || node.lastClick + node.clickLatency < Date.now()) {
+        // because users are used to double-clicking to open files, this prevents a user from opening the same file twice
+
+        node.lastClick = Date.now();
```

Comment:
Any way we can leverage it for all extensions?  Maybe a debounce option for registerCommand?